src/components/auth/Login.tsx

```diff
@@ -39,8 +39,8 @@ const Login = (props: Props) => {
   };
 
   const onSubmit: SubmitHandler<loginDataType> = (data) => {
-    axios
-      .post(`${process.env.REACT_APP_LOGIN_API}/login`, {
+    instance
+      .post(`/login`, {
```

Comment:
We don't need Authentication header for login api call.

Reply:
so instead should I directly use
axios.post(`${process.env.REACT_APP_BASE_URL}/login`